# x_error - absolute value of the x error
# y_error - absolute value of the x error
# theta_error - absolute value of the angular error
float64 x_error
float64 y_error
float64 theta_error